Makefile

```diff
@@ -188,15 +190,11 @@ test\:build:
 
 .PHONY: test\:ci
 test\:ci:
-	DEFRA_BADGER_MEMORY=true DEFRA_BADGER_FILE=true \
-	DEFRA_CLIENT_GO=true DEFRA_CLIENT_HTTP=true \
-	$(MAKE) test:all
-
-.PHONY: test\:ci-gql-mutations
-test\:ci-gql-mutations:
-	DEFRA_MUTATION_TYPE=gql DEFRA_BADGER_MEMORY=true \
-	DEFRA_CLIENT_GO=true DEFRA_CLIENT_HTTP=true \
-	$(MAKE) test:all
+	@$(MAKE) deps:lens
```

Comment:
question: wouldn't this mean that all matrix instances will now build with lens dependencies (even if they don't need lens deps).

Edit: I noticed now we are actually always going to run all lens tests so I guess it fine, I initially thought the lens test will run separately as a row splice in a matrix. Look at suggestion below in that case.